widget/src/main/java/com/uploadcare/android/widget/utils/ParcelableHelper.kt

```diff
@@ -0,0 +1,25 @@
+package com.uploadcare.android.widget.utils
+
+import android.os.Build.VERSION
+import android.os.Build.VERSION_CODES
+import android.os.Bundle
+import android.os.Parcelable
+
+internal fun <T : Parcelable> Bundle.tryGetParcelableSafely(key: String, clazz: Class<T>): T? =
```

Comment:
I think safely more correct there because is about `how` to get value, so it's adverb not adjective.

But I'd like to name function like `getSupportParcelable` because it sounds more convenient and in Android we have functions like `getSupportFragmentManager`, `getSupportActionBar` and etc